miner/worker.go

```diff
@@ -424,23 +417,25 @@ func (self *worker) commitNewWork() {
 		tstamp = parent.Time().Int64() + 1
 	}
 	// this will ensure we're not going off too far in the future
-	if now := time.Now().Unix(); tstamp > now+4 {
+	if now := time.Now().Unix(); tstamp > now+1 {
```

Comment:
This makes the 'too-far-in-the-future' check stricter, by three seconds (?). What are the drawbacks/benefits from that, from a miner perspective?